Makefile

```diff
@@ -1044,7 +1079,14 @@ OPTIMISE_DEFAULT    := -Os
 LTO_FLAGS           := $(OPTIMISATION_BASE) $(OPTIMISE_DEFAULT)
 
 else ifeq ($(TARGET),$(filter $(TARGET),$(F3_TARGETS)))
-OPTIMISE_DEFAULT    := -O2
+OPTIMISE_DEFAULT    := -Ofast
```

Comment:
This changes the default optimisation for F3, thus making a lot of them overrun the flash size.